src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -1082,10 +1082,4 @@ Non-HTTPS access will be removed in a future version. Consider migrating to 'HTT
     <value>Invalid value for WindowsTargetPlatformMinVersion: {0}</value>
     <comment>{0} is the value supplied </comment>
   </data>
-  <data name="Error_InvalidNuGetAuditModeValue" xml:space="preserve">
```

Comment:
Was this intentionally removed?

Reply:
nope, bad merge conflict resolution. I've got more merge conflicts on this file after merging a different PR. I've never had so many merge conflicts on a resx and .Designer.cs file before. I'm unimpressed with the tooling for not being able to sort these out automatically. I think the changes were in different parts of the file. This feels like the bad old days of exclusive locking on an entire file.